index.js

```diff
@@ -3,13 +3,13 @@
 module.exports = {
   name: require('./package').name,
 
-  init: function() {
+  init: function () {
```

Comment:
```suggestion
  init() {
```

Reply:
We should also modernize the JS itself. I can take a first pass in this PR though.